Lib/test/test_subprocess.py

```diff
@@ -54,6 +54,14 @@
 # Ignore errors that indicate the command was not found
 NONEXISTING_ERRORS = (FileNotFoundError, NotADirectoryError, PermissionError)
 
+ZERO_RETURN_CMD = (sys.executable, '-c', 'pass')
+
+
+def setUpModule():
+    if os.access('/bin/true', os.X_OK):
```

Comment:
Should we try to use `shutil.which("true")` instead or in case `/bin/true` is not present? In my system I only have `/usr/bin/true` for example.

Reply:
sure, why not.  done.